docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md

```diff
@@ -1,211 +1,172 @@
-    - `castShadows` to TRUE.
-
-<!-- -->
-
-> **Note**: In the Scene Tree view, the fields are displayed in a different color (depending on the theme) if they differ from their default values.
-
-Now your environment should look like the one depicted in the [figure](prerequisites.md#the-webots-main-window-splits-into-four-dockable-subwindows-the-scene-tree-view-on-the-left-hand-side-including-a-panel-at-the-bottom-for-editing-fields-values-the-3d-view-in-the-center-the-text-editor-on-the-right-hand-side-and-the-console-at-bottom-of-the-window-note-that-some-of-these-subwindows-have-a-toolbar-with-buttons-the-main-menus-appear-on-the-top-of-the-main-window-the-virtual-time-counter-and-the-speedometer-are-displayed-in-the-right-part-of-the-main-toolbar-the-status-text-is-displayed-in-the-bottom-left-of-the-main-window).
-
-> **Hands on**: Save the new world into your project by selecting the `File / Save World As...` menu item.
-Using the dialog box save the world into the "my\_webots\_projects/tutorials/worlds/my\_first\_simulation.wbt" file location.
-
-<!-- -->
+### Create a New World
 
-> **Hands on**: Reload the simulation by selecting the `File / Reload World` menu item.
+A **World** is a file containing information like where the objects are, what they look like, how they interact with each other, what is the color of the sky, how is defined the gravity, friction, masses of the objects, etc.
+It defines the initial state of a simulation.
+The different objects are called **Nodes** and are organized hierarchically in a **Scene Tree**.
+Therefore, a node may contain sub-nodes.
+A world is stored in a file having the `.wbt` extension.
+The file format is derived from the **VRML97** language, and is human readable.
+The world files must be stored directly in a directory called `worlds`.
 
-<!-- -->
+> **Hands on #2**: Pause the current simulation by clicking on the `Pause` button of the 3D view.
+The simulation is paused if the virtual time counter on the main toolbar is stopped.
+Create a new project from the `Wizards` menu by selecting the `New Project Directory...` menu item.
+Follow the instructions.
+Name the project directory `my_first_simulation` instead of the proposed `my_project`.
+Name the world file `my_first_simulation.wbt` instead of the proposed `empty.wbt`.
+Click all the tick boxes, including the "Add a rectangle arena" which is not ticked by default.
 
-> **Note**: You can change the viewpoint of the 3D view by using the mouse buttons (left button, right button and the wheel).
+Webots displays a list of directories and files it just created.
+This corresponds to the [standard file hierarchy of a Webots project](the-standard-file-hierarchy-of-a-project.md).
 
-<!-- -->
+Congratulation, you just created your very first Webots world!
+The 3D view should display a square arena with a checkered floor.
+You can change the viewpoint of the 3D view by using the mouse buttons (left button, right button and the wheel).
```

Comment:
We should link to https://cyberbotics.com/doc/guide/the-3d-window?version=fix-tutorial-1#navigation-in-the-scene